src/views/v2/event-status/postponed-label.php

```diff
@@ -0,0 +1,28 @@
+<?php
+/**
+ * Reason container for a postponed event.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/events-status/postponed-label.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link    http://evnt.is/1aiy
+ *
+ * @version TBD
+ *
+ * @var \WP_Post      $event         The event post object with properties added by the `tribe_get_event` function.
+ * @var Status_Labels $status_labels An instance of the statuses handler.
+ *
+ * @see     tribe_get_event() For the format of the event object.
+ */
+namespace Tribe\Events\Event_Status;
+
+if ( 'postponed' !== $event->event_status ) {
+	return;
+}
+
+?>
+<span class="tribe-events-status-label tribe-events-status-label--postponed">
+	<?php echo esc_html( $status_labels->get_postponed_label() ); ?>
+</span>
```

Comment:
could we use the same template as the canceled label? the only difference it seems is the label and the status type?

Reply:
I would prefer not to, it is easier for someone to overwrite a template per status.

Reply:
From a user perspective, I'd agree here.